<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>L5RCharacterSheetDialog</class>
 <widget class="QDialog" name="L5RCharacterSheetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item row="15" column="5">
    <widget class="QLabel" name="label_25">
     <property name="text">
      <string>Vide:</string>
     </property>
    </widget>
   </item>
   <item row="30" column="12" colspan="5">
    <widget class="QTextEdit" name="m_defaut"/>
   </item>
   <item row="9" column="6">
    <widget class="QLabel" name="label_12">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(150, 150, 255);</string>
     </property>
     <property name="text">
      <string>Intuition</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6" colspan="3">
    <widget class="QLabel" name="label_29">
     <property name="text">
      <string>Gloire:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="3">
    <widget class="QLineEdit" name="m_name"/>
   </item>
   <item row="12" column="9">
    <widget class="QLineEdit" name="m_intell"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(233, 185, 110);</string>
     </property>
     <property name="text">
      <string>Earth:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="3">
    <widget class="QLineEdit" name="m_status"/>
   </item>
   <item row="11" column="6">
    <widget class="QLabel" name="label_13">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 41, 41);</string>
     </property>
     <property name="text">
      <string>Agilité</string>
     </property>
    </widget>
   </item>
   <item row="15" column="6">
    <widget class="QLineEdit" name="m_vide"/>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Clan:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_9">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>Eau</string>
     </property>
    </widget>
   </item>
   <item row="20" column="0" rowspan="9" colspan="11">
    <widget class="QTextEdit" name="m_comps"/>
   </item>
   <item row="4" column="6">
    <widget class="QLabel" name="label_30">
     <property name="text">
      <string>PV:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="9">
    <widget class="QLineEdit" name="m_intuition"/>
   </item>
   <item row="4" column="3" colspan="3">
    <widget class="QLineEdit" name="m_honor"/>
   </item>
   <item row="3" column="6" colspan="3">
    <widget class="QLabel" name="label_31">
     <property name="text">
      <string>Souillure:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QLineEdit" name="m_gloire"/>
   </item>
   <item row="29" column="12" colspan="2">
    <widget class="QLabel" name="label_27">
     <property name="text">
      <string>Défaut:</string>
     </property>
    </widget>
   </item>
   <item row="30" column="0" colspan="10">
    <widget class="QTextEdit" name="m_adv"/>
   </item>
   <item row="4" column="9">
    <widget class="QLineEdit" name="m_pv"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_5">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(233, 185, 110);</string>
     </property>
     <property name="text">
      <string>Constitution:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_7">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>Force:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="9">
    <widget class="QLineEdit" name="m_agi"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_32">
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_28">
     <property name="text">
      <string>Honneur:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3" colspan="3">
    <widget class="QLineEdit" name="m_volo"/>
   </item>
   <item row="13" column="6">
    <widget class="QLabel" name="label_15">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 41, 41);</string>
     </property>
     <property name="text">
      <string>Feu</string>
     </property>
    </widget>
   </item>
   <item row="6" column="6">
    <widget class="QLabel" name="label_10">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(150, 150, 255);</string>
     </property>
     <property name="text">
      <string>Air:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="3">
    <widget class="QLineEdit" name="m_clan"/>
   </item>
   <item row="29" column="0" colspan="4">
    <widget class="QLabel" name="label_26">
     <property name="text">
      <string>Avantages</string>
     </property>
    </widget>
   </item>
   <item row="31" column="13" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="11" column="3" colspan="3">
    <widget class="QLineEdit" name="m_force"/>
   </item>
   <item row="8" column="9">
    <widget class="QLineEdit" name="m_reflexes"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_6">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(233, 185, 110);</string>
     </property>
     <property name="text">
      <string>Volonté:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="6">
    <widget class="QLabel" name="label_14">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 41, 41);</string>
     </property>
     <property name="text">
      <string>intelligence</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="label_24">
     <property name="text">
      <string>Compétences:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="3" colspan="3">
    <widget class="QLineEdit" name="m_percep"/>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_8">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>Perception</string>
     </property>
    </widget>
   </item>
   <item row="3" column="9">
    <widget class="QLineEdit" name="m_souillure"/>
   </item>
   <item row="8" column="6">
    <widget class="QLabel" name="label_11">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(150, 150, 255);</string>
     </property>
     <property name="text">
      <string>Reflexes:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3" colspan="3">
    <widget class="QLineEdit" name="m_const"/>
   </item>
   <item row="1" column="12" rowspan="28" colspan="5">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>260</width>
        <height>899</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="38" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Rang Av3</string>
         </property>
        </widget>
       </item>
       <item row="36" column="1">
        <widget class="QLineEdit" name="m_ranga2t"/>
       </item>
       <item row="24" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Rang4</string>
         </property>
        </widget>
       </item>
       <item row="38" column="1">
        <widget class="QLineEdit" name="m_ranga3t"/>
       </item>
       <item row="37" column="0" colspan="2">
        <widget class="QTextEdit" name="m_ranga2d"/>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QTextEdit" name="m_rang2d"/>
       </item>
       <item row="27" column="0" colspan="2">
        <widget class="QTextEdit" name="m_rang4d"/>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QTextEdit" name="m_rang1d"/>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="m_rang3t"/>
       </item>
       <item row="24" column="1">
        <widget class="QLineEdit" name="m_rang4t"/>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="m_rang2t"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Rang3</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Rang2</string>
         </property>
        </widget>
       </item>
       <item row="29" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Rang5</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0" colspan="2">
        <widget class="QTextEdit" name="m_rang3d"/>
       </item>
       <item row="29" column="1">
        <widget class="QLineEdit" name="m_rang5t"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>School:</string>
         </property>
        </widget>
       </item>
       <item row="33" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Rang Av1</string>
         </property>
        </widget>
       </item>
       <item row="36" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Rang Av2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="m_school"/>
       </item>
       <item row="33" column="1">
        <widget class="QLineEdit" name="m_ranga1t"/>
       </item>
       <item row="30" column="0" colspan="2">
        <widget class="QTextEdit" name="m_rang5d"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="m_rang1t"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Rang1</string>
         </property>
        </widget>
       </item>
       <item row="39" column="0" colspan="2">
        <widget class="QTextEdit" name="m_ranga3d"/>
       </item>
       <item row="34" column="0" colspan="2">
        <widget class="QTextEdit" name="m_ranga1d"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="m_earth">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="13" column="3">
    <widget class="QLabel" name="m_water">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="13" column="9">
    <widget class="QLabel" name="m_fire">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="9">
    <widget class="QLabel" name="m_air">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>L5RCharacterSheetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>L5RCharacterSheetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
